pkg/config/model.go

```diff
@@ -84,6 +84,8 @@ type Meta struct {
 
 	// OwnerReferences allows specifying in-namespace owning objects.
 	OwnerReferences []metav1.OwnerReference `json:"ownerReferences,omitempty"`
+
+	Generation int64
```

Comment:
nit: add a comment. This is one of the most core types in the project so its good to have docs, even if its obvious. Also may need a `json` tag?